app/assets/stylesheets/_settings.scss

```diff
@@ -559,7 +559,7 @@ $tooltip-radius: $global-radius;
 // -----------
 
 $topbar-padding: 0.5rem;
-$topbar-background: $light-gray;
+$topbar-background: #032f3e;
```

Comment:
Define this color as a var in the colors section of this file so you can re-use it later if you want!